package/etc/conf.d/conflib/syslog/app-syslog-liveaction_livenx.conf

```diff
@@ -0,0 +1,18 @@
+block parser app-syslog-liveaction_livenx() {
+    channel {
+        rewrite {
+            r_set_splunk_dest_default(
+                index("netops")
+                sourcetype('liveaction:livenx')
+                vendor("liveaction")
+                product("livenx")
+            )
+        };
+    };
+};
+application app-syslog-liveaction_livenx[sc4s-syslog] {
```

Comment:
looks like liveaction is a PROGRAM , can you check sc4s-syslog-pgm topic here, we have to filter based on program macro.